static/js/dashboard.js

```diff
@@ -455,6 +471,41 @@ function updateTimeRangeForPanel(panelIndex) {
 }
 
 
+function updateTimeRangeForAllPanels(filterStartDate,filterEndDate){
```

Comment:
how is this different from updateTimeRangeForPanels

Reply:
Removed the `updateTimeRangeForPanels` function and added a new function `setTimepickerValue`, which sets the time picker when we first load the dashboards.